.github/workflows/win.yml

```diff
@@ -17,7 +17,8 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: |
-            ~/go/pkg/mod              # Module download cache
+            ~\go\pkg\mod
+            ~\AppData\Local\go-build
```

Comment:
The linux syntax above is ok, but for Windows this is not in line with what I found on https://github.com/mvdan/github-actions-golang#how-do-i-set-up-caching-between-builds:
 - the use of backslashes
 - `%LocalAppData%\go-build`

Reply:
For what I see in the [cache examples](https://github.com/actions/cache/blob/main/examples.md), `~\AppData\Local` and `%LocalAppData%` are the same. I think that is just a matter of taste. If you prefer I will switch to:
```
  ~\go\pkg\mod
  %LocalAppData%\go-build
```

Reply:
Yeah better using the most generic one,  %LocalAppData% then